proxy/get_proxy_url.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""CLI tool that returns URL of the proxy for particular zone and version."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
JFYI, These imports are probably only needed for python 2.

Reply:
indeed. removed